wp-content/themes/missilethreat/assets/_scss/components/_tables.scss

```diff
@@ -40,6 +44,12 @@
     color: $color-text-dark-155;
   }
 
+  .aligncenter.wp-block-table,
+  .alignleft.wp-block-table,
+  .alignright.wp-block-table {
+    display: block;
```

Comment:
I *think* this will override the default `display: table`, which has some impacts on other parts of the table. What was the reasoning for changing this?